CHANGELOG.next.asciidoc

```diff
@@ -37,6 +37,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Metricbeat*
 
+- Add new dashboard for Ceph cluster stats
```

Comment:
Please update this with {pull}13216[13216]

Reply:
fixed